### PR TITLE
Implement basic server API calls

### PR DESCRIPTION
**Pull request by leightonchen**
Handles two basic server calls:
GET /api/:propertyId/basicDetails => details of given propertyId
GET /api/photos/:propertyId       => photo urls of given propertyId

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+app.get('/api/:propertyId/basicdetails', (req, res) => {
+  // TODO: Query property database here and return property's basic details
+  // propertyId stored in req.params.propertyId
+  const propertyId = Number(req.params.propertyId);
+  db.getDetails(propertyId)
+    .then((details) => {
+      res.status(200).send(details);
+    })
+    .catch((error) => {
+      res.status(418).send(error); // i'm a teapot
```
Comment (spraguejdev) [inline server/index.js:21]: your a teapot? Line 21.

File: README.md [hunk 2]
```diff
@@ -32,7 +42,6 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 From within the root directory:
 
 ```sh
-npm install -g webpack
 npm install
 ```
```
Comment (spraguejdev) [inline README.md:46]: Great job with the read me! Looks professional and  gives a user directions on how to start the program.

File: server/index.js [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+    });
+});
+
+app.get('/api/photos/:propertyId', (req, res) => {
```
Comment (spraguejdev) [inline server/index.js:25]: I know the code base is not very large here and how do you feel about breaking your server code into the MVC model such that this file would send the req.body to the controller and then send the data to the model and then the model would grab the data from the db and send it back down to the controller and back down to the server and so on.

**Review by spraguejdev — APPROVED** [review]

File: server/index.js (hunk 1 above)
Comment (leightonchen) [reply server/index.js:21]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418

My favorite error response code. :)

File: server/index.js (hunk 3 above)
Comment (leightonchen) [reply server/index.js:25]: In general, the emphasis of this portion isn't the backend, so I chose not to focus too much on the server-side. Something simple that handles the api calls I need was enough for me. Good practice to do MVC though! If you want me to make those changes, I can do it.

**Pull request merged**